src/util/mirageServer.js

```diff
@@ -15,6 +15,38 @@ export function makeServer() {
       // <-- FIX
 
       this.get('http://localhost:8000/api/v1', { data: 'This is a music app' }),
+        // route to submit user registration form
```

Comment:
Maybe add a little comment to the top of this file directing developers to uncomment any routes that they need/want to mock

Reply:
I will add the comment